charts/artifact-hub/templates/db_migrator_install_job.yaml

```diff
@@ -27,10 +27,10 @@ spec:
         imagePullPolicy: {{ .Values.pullPolicy }}
         env:
           - name: TERN_CONF
-            value: /home/db-migrator/.cfg/tern.conf
+            value: /artifacthub/.cfg/tern.conf
         volumeMounts:
           - name: db-migrator-config
-            mountPath: "/home/db-migrator/.cfg"
+            mountPath: "/artifacthub/.cfg"
```

Comment:
I wasn't expecting the default path where the configuration is mounted to be updated. Please let's make this configurable from the chart leaving the default values as they were (`/home/CMD/.cfg`). We should indicate in the values schema that the only valid values for `configPath` are the values supported (`/home/CMD/.cfg` and `/artifacthub/.cfg`) using an [enum](https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values) for each process.

This will allow you setting the value you need for your use case and it'll be easy to add support for more alternative paths when other similar problems arise.

Reply:
Done.